padle/padle.py

```diff
@@ -258,27 +273,21 @@ async def stats(self, ctx):
                 wins += 1
             elif day != await self.config.num_days() or await self.config.user(ctx.author).done():
                 cur_streak = 0
-            if cur_streak > max_streak:
-                max_streak = cur_streak
-        all_monsters_guessed = []
-        for key, value in all_guesses.items():
-            all_monsters_guessed.extend(value)
+            max_streak = max(max_streak, cur_streak)
+        flatten = itertools.chain.from_iterable
+        all_monsters_guessed = list(flatten(all_guesses.values()))
         if not all_monsters_guessed:
             all_monsters_guessed.append(self.FALLBACK_PADLE_MONSTER)
         mode = max(set(all_monsters_guessed), key=all_monsters_guessed.count)
```

Comment:
This `max` is O(nm), where n is the size of the list and m is the number of unique monsters in it. For each unique monster, you're going through the entire list to count it. It's not a big deal here (10k is small), but be aware.

Prefer a `collections.Counter` when you're trying to count up a lot of things at the same time.